import-export-cli/utils/fileIOUtils.go

```diff
@@ -304,6 +304,38 @@ func CopyDir(src string, dst string) (err error) {
 	return
 }
 
+// CopyDirectoryContents recursively copies all the content of a directory, attempting to preserve permissions.
+// Source directory must exist,and the destination directory exist.
+func CopyDirectoryContents (src string, dst string) (err error)  {
+	entries, err := ioutil.ReadDir(src)
+	if err != nil {
+		return
+	}
+
+	for _, entry := range entries {
+		srcPath := filepath.Join(src, entry.Name())
+		dstPath := filepath.Join(dst, entry.Name())
+
+		if entry.IsDir() {
+			err = CopyDir(srcPath, dstPath)
+			if err != nil {
+				return
+			}
+		} else {
+			// Skip symlinks.
+			if entry.Mode()&os.ModeSymlink != 0 {
+				continue
```

Comment:
Should we be skipping symbolic links? If someone has defined symbolic links inside a params folder for example, it is something that has been done deliberately. In the case where we are copying so that it can be imported, we should resolve these symbolic links and copy over the original file[1]

[1] https://stackoverflow.com/questions/18062026/resolve-symlinks-in-go